data/languages.ts

```diff
@@ -0,0 +1,92 @@
+const languages = [
+  {
+    language: "en",
+    translations: {
+      Register: "Register",
+      Login: "Login",
+      "Create Task": "Create Task",
+      "Users List": "Users List",
+      "Task List": "Task List",
+      Logout: "Logout",
+      Lang: "Lang",
+      "To Do": "To Do",
+      "In Progress": "In Progress",
+      Completed: "Completed",
+      "New Task has been added": "New Task has been added",
+      "Clik here": "Click here",
+      "to the Task list": "to the task list",
+      "Task Details": "Task Details",
+      Title: "Title",
+      Description: "Description",
+      Type: "Type",
+      Status: "Status",
+      "--select status--": "--select status--",
+      assignedTo: "assignedTo",
+      "Created On": "Created On",
+      Cancel: "Cancel",
+      Save: "Save",
+      Edit: "Edit",
+      Tasks: "Tasks",
+      Delete: "Delete",
+      "No tasks": "No Tasks",
+      username: "username",
+      roles: "roles",
+      "No users": "No users",
+      "User Details": "User Details",
+      "User name": "User name",
+      "First name": "First name",
+      "Last name": "Last name",
+      Roles: "Roles",
+      "Add role to the user": "Add role to the user",
+      Manager: "Manager",
+      "User-Default": "User-Default",
+      "Add Role": "Add Role",
+    },
+  },
+  {
+    language: "lv",
+    translations: {
+      Register: "Reģistrēties",
+      Login: "Pieteikties",
+      "Create Task": "Izveidot Uzdevumu",
+      "Users List": "Lietotāju Saraksts",
+      "Task List": "Uzdevumu Saraksts",
+      Logout: "Izrakstīties",
+      Lang: "Valoda",
+      "To Do": "Darāms",
+      "In Progress": "Procesā",
+      Completed: "Pabeigts",
+      "New Task has been added": "Jauns uzdevums ir pievienots",
+      "Click here": "Klikšķiniet šeit",
+      "to the Task list": "uz uzdevumu sarakstu",
+      "Task Details": "Uzdevuma Detaļas",
+      Title: "Nosaukums",
+      Description: "Apraksts",
+      Type: "Tips",
+      Status: "Statuss",
+      "--select status--": "--izvēlieties statusu--",
+      assignedTo: "Piešķirts",
+      "Created On": "Izveidots",
+      Cancel: "Atcelt",
+      Save: "Saglabāt",
+      Edit: "Rediģēt",
+      Tasks: "Uzdevumi",
+      Delete: "Dzēst",
+      "No tasks": "Nav uzdevumu",
+      username: "lietotājvārds",
+      roles: "lomas",
+      "No users": "Nav lietotāju",
+      "User Details": "Lietotāja Detaļas",
+      "User name": "Lietotājvārds",
+      "First name": "Vārds",
+      "Last name": "Uzvārds",
+      Roles: "Lomas",
+      "Add role to the user": "Pievienot lomu lietotājam",
```

Comment:
language json has incorrect entries, usually translation keys with multiple words are separated by a period, so something like this:

user.name

add.role.to.user

Reply:
```
Server   Error: INVALID_KEY: Namespace keys can not contain the character "." as this is used to express nesting. Please remove it or replace it with another character.

Invalid keys: Create.Task (at translations), Users.List (at translations), Task.List (at translations), To.Do (at translations), In.Progress (at translations), New.Task.has.been.added (at translations), Click.here (at translations), to.the.Task.list (at translations), Task.Details (at translations), --select.status-- (at translations), Created.On (at translations), No.tasks (at translations), No.users (at translations), User.Details (at translations), User.name (at translations), First.name (at translations), Last.name (at translations), Add.role.to.the.user (at translations), Add.Role (at translations)
```

this error was preventing to write . between word . so i had to do _ between words